clay/gp.py

```diff
@@ -199,6 +199,14 @@ def from_data(cls, data, source, many=False):
 
         raise AssertionError()
 
+    def inc_playcount(self):
+        if gp.is_subscribed:
```

Comment:
This is more of a question for @ValentijnvdBeek, but I noticed this same pattern for getting the stream URL, though I'm not sure if it's correct. I don't subscribe to Play Music, but just because someone is subscribed doesn't mean the music is from the store. A large portion of my library does not exist in any of the streaming services I've seen.

Reply:
You are right but it is more that name `store_id` is kinda misnomer since Clay used to only be able to play songs from the gmusic library or store. Currently `store_id` has the normal ID (which is what uploaded songs) get as a fallback value. This is not super great but the gmusic API has five different IDs and how they all interact with each other is kinda mystery. 
`self.store_id = (data['storeId'] if 'storeId' in data else data.get('id'))`

It is probably a better idea to use `track.id` instead (which exists) and to improve that interface in the next release

Reply:
I just copy-pasted that part over...

`track.id` sounds good:
> Params song_id: | a song id. Providing the id of a store track that has been added to the library will not increment the corresponding library song’s playcount. To do this, use the ‘id’ field (which looks like a uuid and doesn’t begin with ‘T’), not the ‘nid’ field. (https://unofficial-google-music-api.readthedocs.io/en/latest/reference/mobileclient.html#gmusicapi.clients.Mobileclient.increment_song_playcount)

Reply:
My point was that you're better of replacing it with a `track.id` call since it already exists within the Track class. I am not surprised you didn't notice it because it is somewhat hard to spot in this (somewhat) monolithic version of this file. 

I promise you it got a lot better in the other branch